flask_restful/reqparse.py

```diff
@@ -42,6 +42,8 @@ class Argument(object):
         returned by :meth:`~reqparse.RequestParser.parse_args()`.
     :param bool required: Whether or not the argument may be omitted (optionals
         only).
+    :param bool nullable: Whether or not the argument may be None type (optionals
```

Comment:
@joshfriend It said @huxuan in this issue https://github.com/flask-restful/flask-restful/issues/432 I don't think that it's an required error, required parameter works well. I'm only added new parameter - nullable.